content/content.js

```diff
@@ -176,7 +176,20 @@ var getTranslations = async (image, coordinates, api, idToken, source_lang, targ
           x2: translation['bounding_box'][0] + translation['bounding_box'][2] + coordinates.x,
           y2: translation['bounding_box'][1] + translation['bounding_box'][3] + coordinates.y,
         };
-        showTranslationDialog(translation.translation, ith_coordinates, translation.original, translation.pronunciation, "overlay" + i, true);
+        showTranslationDialog(translation.translation + "\n\n retrieving in-depth translation", ith_coordinates, translation.original, translation.pronunciation, "overlay" + i, true);
+        callTranslateWithText(translation.original, "gpt", idToken, source_lang, target_lang, pronunciation)
+        .then(response => {
+          if (response.error) {
```

Comment:
If there is an error, continue to show the translation from the first response, appended with "\n\n + in-depth translation failed"